test/framework/multicluster.go

```diff
@@ -32,7 +32,8 @@ func NewMulticlusterE2ETest(t *testing.T, managementCluster *ClusterE2ETest, wor
 	for _, c := range workloadClusters {
 		c.clusterFillers = append(c.clusterFillers, api.WithManagementCluster(managementCluster.ClusterName))
 		c.ClusterName = m.NewWorkloadClusterName()
-		c.ClusterConfigLocation = filepath.Join(managementCluster.ClusterConfigFolder, c.ClusterName+"-eks-a.yaml")
+		c.ClusterConfigFolder = c.ClusterName
+		c.ClusterConfigLocation = filepath.Join(c.ClusterConfigFolder, c.ClusterName+"-eks-a.yaml")
```

Comment:
This was not necessary since with the other fixes you can use a custom config folder
However, for consistency with other multicluster tests that set the workload cluster name through the test constructor, I change the config folder here as well to match the same pattern.